modules/iam-user/outputs.tf

```diff
@@ -83,6 +83,7 @@ output "iam_access_key_status" {
 
 output "pgp_key" {
   description = "PGP key used to encrypt sensitive data for this user (if empty - secrets are not encrypted)"
+  sensitive   = true
   value       = var.pgp_key
```

Comment:
`pgp_key` is input into a module and I don't think it makes a lot of sense to mark it sensitive here. It makes sense to mark `sensitive = true` only resource attributes (see line 46 in this file).

Reply:
hmmm I agree with you, although I am getting `Error: Output refers to sensitive values.` when I am trying to use it. (Terraform 0.14.3 / Terragrunt v0.28.19)

Reply:
Nevermind, it seems to be not related to this module itself, but the fact I am creating a new module and using this one as one of the sources....